src/plugins/maps_legacy/public/map/service_settings.test.js

```diff
@@ -59,15 +60,35 @@ describe('service_settings (FKA tile_map test)', function () {
     },
   };
 
+  const mapConfigWithFakeManifest = {
```

Comment:
nit: if order is not important, maybe we could do 

```
const mapConfigWithFakeManifest = {
    ...defaultMapConfig,
    emsFakeManifestApiUrl,
  };
```
to remove some duplications?